app/Http/Controllers/OpeninghoursController.php

```diff
@@ -142,8 +136,6 @@ public function destroy(DeleteOpeninghoursRequest $request, $id)
         $success = $this->openinghours->delete($id);
 
         if ($success) {
-            event(new OpeninghoursDeleted($openinghours, $this->openinghours->isOpeninghoursRelevantNow($openinghours)));
-
             return response()->json(['message' => 'De openingsuren werden verwijderd']);
```

Comment:
Make message translatable

Reply:
will be done in issue  https://github.com/StadGent/laravel_site_opening-hours/issues/85